add microgrant post (#229)
open-life-science/open-life-science.github.io: _posts/2020-11-23-micro-grants-in-2021-ols-3-ols-4.md

- bebatut
- malvikasharan
- yochannah
image: https://images.unsplash.com/photo-1579621970563-ebec7560ff3e?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=1951&q=80
photos:
  name: Photo by Micheile Henderson on Unsplash
  url: https://unsplash.com/@micheile
---

Open Life Science has recently been lucky enough to be awarded two small grants - one from the EOSC-Life project, which will partially support OLS-3, and two more small grants which will partially support OLS-3 and 4. In all cases, the funds awarded are less than the full cost of a cohort if we were to pay for all the things we would like to pay for, including mentor and expert speaker time, but nevertheless - it is a helpful start, and one we are privileged to have.

For OLS-3 and OLS-4 we are delighted to announce that all project leaders (aka mentees) - will be able to request small seed grants for expenses that they may need for their participation in OLS or to run their project. This could be an audio headset, the cost for more data on a tethered mobile phone to attend calls, fliers or stickers with your project logo on it, or the fee to register your project as a legal organisation in your country.

> ## How much?

Reviewer: ```suggestion
## How much?

```

Given that our current budget is very small - around $5000 USD in total for OLS-3 and OLS-4 - our cap for eligible expenses will be $50 USD for personal expenses such as high speed internet or a new headset, and up to $200 in total for project-specific expenses such as company registration or stickers. If we receive more micro-grant requests than we are able to cover, we will seek additional funding but may not be able to guarantee we can cover beyond the initial $2500 budget for each cohort. If your employer is able to cover a given expense, we request that they do so. In order to make the most impact possible, this fund is intended for those who might not be able to afford the expense, otherwise.

> ## How can I apply?

Reviewer: ```suggestion
## How can I apply?

```

> In order to be eligible, you must be an accepted member of Open Life Science as a project leader, aka a mentee, who is actively participating in the program. If you’re a mentor or expert speaker for OLS (any round), or if you’re a graduate from a previous cohort, we may be able to support similar expenses separately - please contact team@openlifesci.org.

Reviewer 2: Do we need to add what happens if they drop out after 1st month? or not complete the program?

Author: I am not sure what there is we can do here, but open to options...

Reviewer 2: No, no idea as well. :/ there is no way we can control the expense if people make use of them and drop off.

Author: I guess that is part of the reason for having the 4-week limit before project expenses are eligible and the requirement that participants be active, as well as having a cap on how much people can ask for, at least early on?

Reviewer 2: Good point.

> In order to be eligible, you must be an accepted member of Open Life Science as a project leader, aka a mentee, who is actively participating in the program. If you’re a mentor or expert speaker for OLS (any round), or if you’re a graduate from a previous cohort, we may be able to support similar expenses separately - please contact team@openlifesci.org.

Reviewer: ```suggestion
In order to be eligible, you must be an accepted member of Open Life Science as a project leader, aka a mentee, who is actively participating in the program. If you're a mentor or expert speaker for OLS (any round), or if you're a graduate from a previous cohort, we may be able to support similar expenses separately - please contact [{{ site.email|replace:'@','[at]' }}](mailto:{{ site.email }}).
```

> ## What is the timeline?

Reviewer: ```suggestion
## What is the timeline?

```

Applications for OLS-3 will open on December 1st 2021, and applications for OLS-4 will open mid 2021.

> 

Reviewer: ```suggestion
## What can I apply for?

```

> ### Personal expenses:

Reviewer: ```suggestion
### Personal expenses

```

OLS project leads will be eligible to request personal expenses as soon as they are notified of acceptance to OLS-3 or OLS-4.

> ### Project expenses:

Reviewer: ```suggestion
### Project expenses

```

Project-specific costs will be eligible from week 4 of the currently-running OLS cohort onwards, providing that the project lead has been participating actively in the program - i.e. attending mentor calls and communicating on Slack or GitHub, and attending at least half of the cohort calls. We do not expect participants to attend all cohort calls as this is a global program that spans many time zones, but we do expect participants to catch up on missed calls via the OLS YouTube Channel.

> ### Week 12 onwards:

Reviewer: ```suggestion
### Week 12 onwards

```

> After week 12 of the program, if the full budget for the cohort has not yet been spent, we may open up requests to larger expenses than our previously stated caps.

Reviewer 2: We might also be able to carry over the left over to the next cohort.

> After week 12 of the program, if the full budget for the cohort has not yet been spent, we may open up requests to larger expenses than our previously stated caps.

Author: ```suggestion
After week 12 of the program, if the full budget for the cohort has not yet been spent, we may open up requests to larger expenses than our previously stated caps. Any unspent budget will roll over to the next cohort.
```

> ## How will it work?

Reviewer: ```suggestion
## How will it work?

```

---
layout: post
title: OLS Community support micro grants
authors:
- bebatut
- malvikasharan
- yochannah
image: https://images.unsplash.com/photo-1579621970563-ebec7560ff3e?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=1951&q=80
photos:
  name: Photo by Micheile Henderson on Unsplash
  url: https://unsplash.com/@micheile
---

> Open Life Science has recently been lucky enough to be awarded two small grants - one from the EOSC-Life project, which will partially support OLS-3, and two more small grants which will partially support OLS-3 and 4. In all cases, the funds awarded are less than the full cost of a cohort if we were to pay for all the things we would like to pay for, including mentor and expert speaker time, but nevertheless - it is a helpful start, and one we are privileged to have.

Reviewer 2: ```suggestion
Open Life Science has recently been awarded three grants - one from the EOSC-Life project, which will partially support OLS-3, and two more small grants which will partially support OLS-3 and 4. The full cost of these two cohorts are higher than the current budget if we were to pay for all the things we would like to pay for, including mentor and expert speaker time, but nevertheless - it is a significant start and one we are privileged to have.
```

Reviewer 2: I think we don't need to add this here "The full cost of these two cohorts are higher than the current budget if we were to pay for all the things we would like to pay for, including mentor and expert speaker time, but nevertheless - it is a significant start and one we are privileged to have."

> For OLS-3 and OLS-4 we are delighted to announce that all project leaders (aka mentees) - will be able to request small seed grants for expenses that they may need for their participation in OLS or to run their project. This could be an audio headset, the cost for more data on a tethered mobile phone to attend calls, fliers or stickers with your project logo on it, or the fee to register your project as a legal organisation in your country.

Reviewer 2: ```suggestion
With this development, we are delighted to announce that all OLS-3 and OLS-4 project leaders (aka mentees) - will be able to request small seed grants for expenses that they may need for their participation in OLS or to run their project. This could be an audio headset, the cost for more data on a tethered mobile phone to attend calls, fliers or stickers with your project logo on it, or the fee to register your project as a legal organisation in your country.
```
